Makefile

```diff
@@ -181,7 +181,7 @@ ensure-yarn: # Yarn ensures the correct version of yarn is installed
 
 yarn:
 	corepack enable
-	corepack prepare yarn@stable --activate
```

Comment:
changed this because corepack complains about using `stable`:
`Invalid package manager specification in CLI arguments; expected a semver version`